OrbitGl/CaptureWindow.cpp

```diff
@@ -515,6 +515,35 @@ void CaptureWindow::Draw() {
     // Vertical green line at mouse x position
     ui_batcher_.AddVerticalLine(pos, -world_height_, kZValueText, Color(0, 255, 0, 127));
   }
+  DrawScreenSpace();
+
+  // We start by getting all layers
+  auto layers = time_graph_.GetBatcher().GetLayers();
+  for (auto layer : ui_batcher_.GetLayers()) {
+    layers.insert(layer);
+  }
+  for (auto layer : text_renderer_.GetLayers()) {
+    layers.insert(layer);
+  }
+  for (auto layer : time_graph_.GetTextRenderer()->GetLayers()) {
+    layers.insert(layer);
+  }
+
+  const float kScreenSpaceMinimumLayer = 0.8f;
+  for (float layer : layers) {
+    // We use different coordinate system for ScreenSpace items (margin, scrollbar, ...)
```

Comment:
Some typos here. Replace with:

// We use different coordinate systems for ScreenSpace items (margin, scrollbar, ...)
// and for the text than the rest of items inside CaptureWindow. So, we have to switch
// between these 2 systems while the layer is changing (with these "Prepare.." functions).